tests/test_utils_commond.py

```diff
@@ -37,11 +38,5 @@ def test_paramiko_fail_connection(ssh_server, mock_post_request, base_url, cmds_
 
     command = load_template_values(validate_schema(cmds_path_config), "test_post_command", dict())
 
-    try:
+    with pytest.raises((ssh_exception.NoValidConnectionsError, ssh_exception.SSHException)):
```

Comment:
which exception do you expect here ? .... NoValidConnectionError or SSHException?

Reply:
I am expecting any of them.

Reply:
how so ? ... if either one could be thrown, then this is a test that depends on some random external behaviour

Reply:
mmm, my test is expecting an error where paramiko is unable to connect with the default port in the mock server.